polar2grid/glue.py

```diff
@@ -47,6 +47,8 @@
 except ImportError:
     CRS = None
 
+os.environ["PPP_CONFIG_DIR"] = os.path.join(sys.prefix, 'etc', 'satpy')
```

Comment:
You may want to use `os.path.dirname(__file__)` or something similar. Maybe pkg_resources has a way to get the path of the `etc/satpy` directory?

What do you think about changing this to `etc/polar2grid` too? I know this would require find/replacing it everywhere that it is used but now that this will be installed this way any time polar2grid is installed, it would be best to avoid satpy's configuration (right now this doesn't because of how satpy is set up, but we should be careful for future changes).

Reply:
I have no preference for the naming. I feel like naming is satpy makes sense since it's data for satpy, but I'm fine with polar2grid. The problem with os.path.dirname(__file__) is that the file will be in site_packages, while the config directory is at the base level. I guess I could manually have it traverse back some levels, but that feels like an error waiting to happen.

Reply:
Agreed on the error waiting to happen. See if the `pkg_resources` package can be used to tell you where the `etc` directory is. `pkg_resources` should know where `setup.py` installed things.

Altlhough these configs get used by Satpy, we are placing them in `<prefix>/etc/satpy` which is a "global" location that may be used by satpy. Only Satpy (and optionally satpy plugins in the future) should be modifying things in `etc/satpy`. Note that Satpy doesn't currently use `etc/satpy` but that may change in the future. This is my thinking at least.